test/test_ops.py

```diff
@@ -78,8 +78,14 @@ def _check_helper(self, device, dtype, op, variant, check):
         if not op.supports_dtype(dtype, torch.device(device).type):
             self.skipTest(f"Skipped! {op.name} does not support dtype {str(dtype)}")
 
+        def is_inplace(variant):
+            return variant.__name__.endswith('_')
```

Comment:
Is there a better way to check this?

Or should we trickle variant type and variant from the function that calls this helper?

https://github.com/pytorch/pytorch/blob/c4c77e2001ead838932cab61c5813c25c4f3e72e/test/test_ops.py#L141
https://github.com/pytorch/pytorch/blob/c4c77e2001ead838932cab61c5813c25c4f3e72e/test/test_ops.py#L161

Reply:
The correct way to check this is by testing if the variant `is` the inplace variant acquired from the OpInfo.

Reply:
Unfortunately that won't work as the `op.get_inplace()` is wrapped by `_get_safe_inplace`.

https://github.com/pytorch/pytorch/blob/c4c77e2001ead838932cab61c5813c25c4f3e72e/test/test_ops.py#L68-L73

Reply:
Can you access it through the dunder wrapped attribute?

```
foo.__wrapped__ is op.get_inplace()
```

Reply:
Yup it works with that. Thanks!!

```python
def is_inplace(variant):
    if hasattr(variant, "__wrapped__"):
      return variant.__wrapped__ is op.get_inplace()

    return variant is op.get_inplace()
```